packages/client/hooks/useDraggableReflectionCard.tsx

```diff
@@ -33,19 +33,37 @@ const useRemoteDrag = (
   staticIdx: number
 ) => {
   const setPortal = useContext(PortalContext)
-  const {remoteDrag, isDropping} = reflection
+  const {remoteDrag, isDropping, reflectionGroupId} = reflection
   const setRemoteCard = (isClose: boolean, timeRemaining: number, lastTop?: number) => {
     if (!drag.ref || timeRemaining <= 0) return
     const beforeFrame = Date.now()
     const bbox = drag.ref.getBoundingClientRect()
     if (bbox.top !== lastTop) {
+      const targetId = remoteDrag?.targetId
+      const isTargetInSpotlight = !!document.querySelector(
```

Comment:
@mattkrick I looked into using a ref instead of the `querySelector`, but I think we need to have the logic within `setRemoteCard` as it's recursively updating with every animation frame. In `DraggableReflectionCard.tsx`, we could use the `targetId` to grab the target element and store it in `drag`,  but it won't update as quickly as we need

Reply:
thanks for digging in!
inside hooks it should be AOK, if there's a better way to do it inside the a functional component let's try that. trying to access the DOM while computing what the vdom should look like might get dicey

possible ideas:

- `useState(() => document.querySelector(...))`
- `useRef` + `useEffect(() => { if (blah blah) document.querySelector}, [])`